src/main/java/com/alkemy/ong/dto/TestimonialDto.java

```diff
@@ -4,12 +4,21 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
 public class TestimonialDto {
 
+    @NotBlank(message = "the name field cannot be empty")
+    @NotNull(message = "field name cannot be null")
```

Comment:
shouldn't this go straight in the entity? I understood so

Reply:
I decided to put them here. :)